lib/plugins/aws/package/lib/validateCompiledTemplate.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const CF = require('aws-sdk/clients/cloudformation');
```

Comment:
I would not use the aws-sdk directly here. SLS has an abstraction of the calls in the provider class (this.provider).
Like this:
```
    return this.provider.request(
      'CloudFormation',
      'validateTemplate',
      params,
      this.options.stage,
      this.options.region
    ).then((cfData) => this.monitorStack('create', cfData));
```

This will automatically let you monitor the operation and display it nicely as well as it works with regions and stages.
`this.provider` is available in every plugin.